src/main/kotlin/org/oso/AppInitialData.kt

```diff
@@ -24,7 +30,10 @@ class AppInitialData
         LOGGER.info("Creating initial data...")
 
         // ensure that the objects are only initialized once
-        if (deviceRepository.count() > 0 && deviceTypeRepository.count() > 0) {
+        if (helpRequesterRepository.count() > 0 &&
```

Comment:
The database should initialize his testdata by his own but we also can do this later.

There are a few reasons, for example:
- Service A has instances of different versions deployed (could make trouble in this way)
- In an bad case this could inject not wanted testdata in our production environment.
- Our Services should have the same behaviour in every environment (production, staging, testing)
- Configurations have to be injected from external into the application (Twelve-Factor App methodology)
- this would bloated the code

Reply:
Sure no problem. This is just a temporary solution for our upcoming project showing. 😀